aws/datalake/blueprint.yaml

```diff
@@ -15,12 +15,15 @@ spec:
 - name: AdministratorName
   description: Name of the Data Lake administrator
   type: Input
+  pattern : ".+"
```

Comment:
@deepu105 Can you also add validation for S3BucketName? You may see name requirements [here](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/cloudtrail-s3-bucket-naming-requirements.html)

Reply:
ok

Reply:
its harder than I thought coz backreference and lookahead doesnt work in golang